File: server/src/main/java/com/darly/db/entity/record/Record.java
package com.darly.db.entity.record;

import com.darly.db.entity.user.User;
import lombok.*;
import org.hibernate.annotations.DynamicInsert;
import org.hibernate.annotations.OnDelete;
import org.hibernate.annotations.OnDeleteAction;

import javax.persistence.*;

@Entity
@Getter
@Setter
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
@DynamicInsert
@Builder
@Table(name = "tb_record")
public class Record {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long recordId;
    @ManyToOne
    @JoinColumn(name = "user_id")
    @OnDelete(action = OnDeleteAction.CASCADE)
    private User user;
    private Long dayId;
    private Long matchId;
    private String recordTitle;
    private Long recordDate;
    private Float recordDistance;
    private Long recordTime;
    private Integer recordPace;
    private Float recordSpeed;
    private Integer recordHeart;
    private Integer recordCalories;
    private Integer recordRank;
    private String recordImage;
}

File: Android/app/src/main/java/com/ssafy/darly/model/record/Record.kt
package com.ssafy.darly.model.record

data class Record(
    val recordId: Long,
    val recordTitle: String,
    val recordDate: String,
    val recordDistance: Float,
    val recordTime: Int, //sec
    val recordPace: Float,
    val recordHeart: Int,
    val recordCalories: Int,
    var recordImage: String?,
    val coordinateLatitudes: MutableList<String>,
    val coordinateLongitudes: MutableList<String>,

    var recordDistanceString: String,
    var recordTimeString: String,
    var recordPaceString: String,
    var recordHeartString: String,
    var recordCaloriesString: String,
)